modules/installation-osp-custom-subnet.adoc

```diff
@@ -11,22 +11,24 @@
 
 Optionally, you can deploy a cluster on a {rh-openstack-first} subnet of your choice. The subnet's GUID is passed as the value of `platform.openstack.machinesSubnet` in the `install-config.yaml` file.
 
-This subnet is used as the cluster's primary subnet; nodes and ports are created on it.
+This subnet is used as the cluster's primary subnet. By default, nodes and ports are created on it. You can create nodes and ports on a different {rh-openstack} subnet by setting the value of the `platform.openstack.machinesSubnet` property to the subnet's UUID.
 
 Before you run the {product-title} installer with a custom subnet, verify that:
 
-* The target network and subnet are available.
+* The subnet that is used by `platform.openstack.machinesSubnet` has DHCP enabled.
+* The CIDR of `platform.openstack.machinesSubnet` matches the CIDR of `networking.machineNetwork`.
+* The installer user has permission to create ports on this network, including ports with fixed IP addresses.
 
-* DHCP is enabled on the target subnet.
+Clusters that use custom subnets have the following limitations:
 
-* You can provide installer credentials that have permission to create ports on the target network.
-
-* If your network configuration requires a router, it is created in {rh-openstack}. Some configurations rely on routers for floating IP address translation.
-
-* Your network configuration does not rely on a provider network. Provider networks are not supported.
+* If you plan to install a cluster that uses floating IPs, the `platform.openstack.machinesSubnet` subnet must be attached to a router that is connected to the `externalNetwork` network.
+* The installer cannot create a private network or subnet for your {rh-openstack} machines if the `platform.openstack.machinesSubnet` value is set in the `install-config.yaml`.
```

Comment:
`The installer cannot` or `The installer do not` ?

Reply:
My guess is that "does not" is more accurate. @iamemilio ?

Reply:
yeah, you are right, sorry for my poor english.